pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -1474,6 +1541,7 @@ func testProvision() *hivev1.ClusterProvision {
 
 func testSuccessfulProvision() *hivev1.ClusterProvision {
 	provision := testProvision()
+	provision.ObjectMeta.Labels[jobNameLabel] = install.GetInstallJobName(provision)
```

Comment:
Why would the provision be labelled with the name of the job?

Reply:
My bad, its not needed